doc/conf.py

```diff
@@ -78,8 +78,8 @@
 # The encoding of source files.
 #source_encoding = 'utf-8'
 
-# The master toctree document.
-master_doc = 'contents'
```

Comment:
Are you sure about this change. I don't see any code below using this variable (is sphinx defining `master_doc`)

Reply:
it's used in a different file if I remember correctly